docs/API.md

```diff
@@ -312,5 +412,88 @@ Or `.static()` ...
 Or even `.compose()` ...
 
 ```js
-  var obj = abstractStamp.compose(concreteStamp, additionalStamp, utilityStamp).create();
+  const obj = abstractStamp.compose(concreteStamp, additionalStamp, utilityStamp).create();
+```
+
+
+## Chaining methods
```

Comment:
> One thing at a time Tim. I'm not changing text at this point of time. Just moving.

You have introduced several new examples in this PR, this is not just a "moving" commit. I'm not asking for anything that you have not already started to do already.

> Done

:+1: